### ISSUE TITLE
Tests doesn't pass on go1.10

### ISSUE THREAD

**Issue by DamnWidget** [mitchellh/go-mruby#58]
Hi guys, looks like tests doesn't pass after go1.10 update:

```cp vendor/mruby/build/host/lib/libmruby.a .
Checking code with gofmt..
gofmt -s *.go >/dev/null
sh golint.sh
Installing golint into your GOPATH...
Checking with golint...
go test -v
# github.com/DamnWidget/aim/vendor/github.com/mitchellh/go-mruby
./array_test.go:38: Fatalf format %s has arg valType of wrong type mruby.ValueType
./hash_test.go:33: Fatalf format %s has arg valType of wrong type mruby.ValueType
./hash_test.go:58: Fatalf format %s has arg value.Type() of wrong type mruby.ValueType
./parser_test.go:19: Fatalf format %s has arg warns of wrong type []*mruby.ParserMessage
./parser_test.go:44: Fatalf format %s has arg warns of wrong type []*mruby.ParserMessage
./value_test.go:245: Fatalf format %s has arg cType of wrong type mruby.ValueType
FAIL	github.com/DamnWidget/aim/vendor/github.com/mitchellh/go-mruby [build failed]
make: *** [Makefile:27: test] Error 2```

**Comment by mitchellh** [comment]
Good catch thank you, fixed in #59!

**Comment by erikh** [comment]
<3
